(dp1
S'output'
p2
S"<type 'exceptions.AttributeError'> 'Table' object has no attribute 'frquency'"
p3
sS'layer'
p4
S'/home/abdulhalim/Desktop/web2py/applications/motor/controllers/default.py'
p5
sS'code'
p6
S'<code object <module> at 0x7f2f61b20d30, file "/home/abdulhalim/Desktop/web2py/applications/motor/controllers/default.py", line 2>'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "/home/abdulhalim/Desktop/web2py/gluon/restricted.py", line 219, in restricted\n    exec(ccode, environment)\n  File "/home/abdulhalim/Desktop/web2py/applications/motor/controllers/default.py", line 138, in <module>\n  File "/home/abdulhalim/Desktop/web2py/gluon/globals.py", line 409, in <lambda>\n    self._caller = lambda f: f()\n  File "/home/abdulhalim/Desktop/web2py/gluon/tools.py", line 3964, in f\n    return action(*a, **b)\n  File "/home/abdulhalim/Desktop/web2py/applications/motor/controllers/default.py", line 63, in dev_control\n    db.Control_Instruction.frquency.readable = db.Control_Instruction.frquency.writable = False\nAttributeError: \'Table\' object has no attribute \'frquency\'\n'
p11
s.